cpp/tests/framework/containers/FixedVector.test.cpp

```
@@ -0,0 +1,74 @@
+// Copyright Robotick Labs
+// SPDX-License-Identifier: Apache-2.0
+
+#include "robotick/framework/containers/FixedVector.h"
+#include <catch2/catch_all.hpp>
+
+namespace robotick::test
+{
+	TEST_CASE("Unit/Framework/Common/FixedVector")
```

Clang (14.0.6)

[warning] variable 'TEST_CASE' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)